charts/vela-core/templates/defwithtemplate/task.yaml

```diff
@@ -16,6 +16,7 @@ spec:
       		parallelism: parameter.count
       		completions: parameter.count
       		template: spec: {
+                        restartPolicy: parameter.policy
```

Comment:
The CUE parameter doesn't have a policy field, where is this referring to?  You can just hardcode it to be "Never" here

Reply:
sorry,task.cue has been updated